community/index.md

```diff
@@ -45,66 +45,49 @@ These forums are covered by the [Scala Code of Conduct](../conduct.html).
 
-Job listings are not allowed in our other forums and chat rooms.
+Job postings are not allowed in our other forums and chat rooms.
 
 ## Chat Rooms
 
-Our main chat platform is [Gitter](https://gitter.im).  See next section for other platforms.
+Our main chat platform is Discord, and the main Scala server is:
 
-The main Gitter room for Scala is:
+* **[Scala](https://discord.com/invite/scala)**
+  * the #scala-users channel is especially beginner-friendly
+  * the #scala-contributors channel is about moving Scala forward
+  * the #jobs channel is the only place we allow job postings
+  * ask on #admin if you have questions or suggestions about the server itself
+  * there are many other channels, including #spark, #scala-js, and #scala-native
 
-* **[scala/scala](https://gitter.im/scala/scala)**: Questions, general discussion, etc. Beginner-friendly.
+Scala-oriented Discord servers operated by the community include:
 
-Other, more specialized rooms include:
-
-* **[scala/center](https://gitter.im/scala/center)**: for discussions about Scala community governance, processes, the Scala Platform, and projects going on at the Scala Center.
-* **[scala/contributors](https://gitter.im/scala/contributors)**: for contributors to discuss work on changes to Scala.
-* **[scala/moocs](https://gitter.im/scala/moocs)**: for talking about the Scala Center's online courses
-* **[scala/job-board](https://gitter.im/scala/job-board)**: for employers and job seekers to connect with each other
-* **[scala-native/scala-native](https://gitter.im/scala-native/scala-native)**: for discussion about the Scala to LLVM compiler.
-* **[scala-js/scala-js](https://gitter.im/scala-js/scala-js)**: for discussion about the Scala to JavaScript compiler.
+* **[Scalameta](https://discord.gg/RFpSVth)**: Scalameta-based tooling: Metals, Scalameta, Scalafix, Scalafmt, and Mdoc
+* **[Typelevel](https://discord.gg/XF3CXcMzqD)**: about the Typelevel ecosystem for pure-functional programming in Scala
 
-International rooms are available as well:
+English-language Scala rooms on other chat platforms besides Discord include:
 
-* **[scala/scala-india](https://gitter.im/scala/scala-india)**
-* **[scala/ar](https://gitter.im/scala/ar)**
-* **[scala/cn](https://gitter.im/scala/cn)**
-* **[scala/de](https://gitter.im/scala/de)**
-* **[scala/es](https://gitter.im/scala/es)**
-* **[scala/fr](https://gitter.im/scala/fr)**
-* **[scala/it](https://discord.gg/8wadTgcZVt)** (Discord)
-* **[scala/pl](https://gitter.im/scala/pl)**
-* **[scala/pt](https://gitter.im/scala/pt)**
-* **[scalajp](https://gitter.im/scalajp/)**
-* **[scala_ru](https://t.me/scala_ru)** (Telegram)
 * **[scala_en](https://t.me/scala_en)** (Telegram)
+* **[scala/scala](https://gitter.im/scala/scala)** (Gitter)
+* **[#scala](https://libera.chat)** (IRC)
 
-All of the rooms above are covered by the [Scala Code of Conduct](../conduct.html).
```

Comment:
This sentence seems to have disappeared, but it's important that we keep it.

Reply:
Oops! Yeah, that got lost when I was shuffling stuff around.

I've restored it, but I wasn't quite sure whether to put it all the way at the bottom (since I'm not actually sure if e.g. the Telegram links are under Scala CoC), or just have it cover our server only, or scope it for all the Discord servers only. I went with "our server only".

Reply:
Our servers only is the right thing to say IMO.